cli/Valet/PhpFpm.php

```diff
@@ -297,9 +297,18 @@ private function installExtensions(string $version): void
     {
         $extArray = [];
         $extensionPrefix = $this->pm->getPhpExtensionPrefix($version);
-        foreach (self::COMMON_EXTENSIONS as $ext) {
-            $extArray[] = "{$extensionPrefix}{$ext}";
+
+        if (($this->pm instanceof \Valet\PackageManagers\Pacman)) {
```

Comment:
Same as above, instead of adding if else statement here, can we relay on methods from PackageManager?

Reply:
In this case, i have to leave it as such since ```php``` for pacman automatically ships in with all extensions (except ```php-apache```, ```php-apcu```, ```php-cgi```, ```php-dblib```, ```php-embed```, ```php-enchant```,```php-geoip```, ```php-gd```  which am not sure if they're all php extensions).

The only package needed by valet here is ```php-gd```. Don't know how it works for other packages though